guide/interactions/registering-slash-commands.md

```diff
@@ -161,3 +161,62 @@ const data = {
 	}],
 };
 ```
+
+## Advanced deployment
+
+Advanced users should make use of a deploy script to deploy their commands when needed. In this section we'll be using a script that is usable in conjunction with the [slash command handler](/command-handling) from the command handling section.
+
+First off, install the discord.js REST module by running `npm i @discordjs/rest` in your terminal.
+
+::: warning
+For this script to work you need to make sure you **don't** use string literals for the `ApplicationCommandOptionType` in your command files, but rather the `ApplicationCommandOptionType` enum from `discord-api-types`.
+:::
+
+<!-- eslint-skip -->
+
+```js
+const { REST } = require('@discordjs/rest');
+const { Routes } = require('discord-api-types/v9');
+const { token } = require('./config.json');
+const { readdirSync } = require('fs');
+
+const commands = [];
+
+const commandFiles = readdirSync('./commands').filter(file => file.endsWith('.js'));
+
+for (const file of commandFiles) {
+	const command = require(`./commands/${file}`);
+	const { name, description, options } = command;
+
+	const data = {
+		name,
+		description,
+		options,
+	};
+
+	commands.push(data);
+}
+
+const rest = new REST({ version: '9' }).setToken(token);
+
+try {
+	console.log('Started refreshing application (/) commands');
+
+	await rest.put(
+		Routes.applicationGuildCommands(
+			<CLIENT ID>,
+			<GUILD ID>,
```

Comment:
```suggestion
			<Client>.application.id,
			<Guild>.id,
```

Or

```suggestion
			<Client>.user.id,
			<Guild>.id,
```

Reply:
This makes no sense in context - this section of the guide is using the `@discordjs/rest` module, not `discord.js` and as such Client and Guild classes don't exist.